manifests/charts/gateway/values.yaml

```diff
@@ -129,3 +129,8 @@ imagePullSecrets: []
 podDisruptionBudget: {}
 
 terminationGracePeriodSeconds: 30
+
+# Use RollingUpdate deployment strategy to reduce disruption
+rollingUpdate:
+  maxSurge: 50%
+  maxUnavailable: 25%
```

Comment:
istio tends to take a bit more agnostic approach to this and would prefer admins/operations to make the determination on what to set these values to. Can you templatize this similar to `podDisruptionBudget`?